thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -0,0 +1,303 @@
+import Ember from 'ember';
+import _ from 'lodash';
+
+//
+// Config
+//
+
+const EVENT_TABLE_COLUMNS = [
+  {
+    template: 'custom/checkbox',
+    useFilter: false,
+    mayBeHidden: false,
+    className: 'events-table__column--checkbox'
+  },
+  {
+    propertyName: 'label',
+    title: 'Event Name',
+    className: 'events-table__column'
+  },
+  {
+    propertyName: 'eventType',
+    title: 'Type',
+    filterWithSelect: true,
+    sortFilterOptions: true,
+    className: 'events-table__column--compact'
+  },
+  {
+    propertyName: 'start',
+    title: 'Start',
+    className: 'events-table__column--compact',
+    disableFiltering: true
+  },
+  {
+    propertyName: 'end',
+    title: 'End',
+    className: 'events-table__column--compact',
+    disableFiltering: true
+  }
+];
+
+//
+// Controller
+//
+
+export default Ember.Controller.extend({
+  selectedUrns: null, // Set
+
+  filteredUrns: null, // Set
+
+  invisibleUrns: null, // Set
+
+  _timeseriesCache: null, // {}
+
+  _entitiesCache: null, // {}
+
+  _pendingRequests: null, // {}
+
+  entities: Ember.computed(
+    '_entitiesLoader',
+    '_entitiesCache',
+    function () {
+      console.log('entities()');
+      this.get('_entitiesLoader'); // trigger loader. hacky
+      return this.get('_entitiesCache') || {};
+    }
+  ),
+
+  timeseries: Ember.computed(
+    '_timeseriesLoader',
+    '_timeseriesCache',
+    function () {
+      console.log('timeseries()');
+      this.get('_timeseriesLoader'); // trigger loader. hacky
+      return this.get('_timeseriesCache') || {};
+    }
+  ),
+
+  chartSelectedUrns: Ember.computed(
+    'model.primaryMetricUrn',
+    'selectedUrns',
+    'invisibleUrns',
+    function () {
+      console.log('chartSelectedUrns()');
+      const selectedUrns = this.get('selectedUrns');
```

Comment:
please use `this.getProperties` when retrieving multiple properties

Reply:
fixed